### PR TITLE
#RI-3479 - add redis-auto button

### PR DESCRIPTION
**Pull request by romansergeenkosofteq**
also fix id for plugin iframes

### CODE REVIEW

File: redisinsight/ui/src/pages/workbench/components/enablement-area/EnablementArea/components/CodeButton/CodeButton.tsx [hunk 1]
```diff
@@ -1,32 +1,47 @@
...
+
+  return (
+    <EuiButton
+      iconSide="right"
+      isLoading={isLoading}
+      size="s"
+      onClick={() => onClick(execute, params)}
+      fullWidth
+      color="secondary"
+      className={[className, styles.button].join(' ')}
```
Comment (zalenskiSofteq) [inline redisinsight/ui/src/pages/workbench/components/enablement-area/EnablementArea/components/CodeButton/CodeButton.tsx:28]: change `[...].join` to `cx(...)`

File: redisinsight/ui/src/pages/workbench/components/enablement-area/EnablementArea/EnablementArea.tsx [hunk 2]
```diff
@@ -25,7 +26,13 @@ export interface Props {
   guides: Record<string, IEnablementAreaItem>
   tutorials: Record<string, IEnablementAreaItem>
   loading: boolean
-  openScript: (script: string, path?: string, name?: string) => void
+  openScript: (
```
Comment (zalenskiSofteq) [inline redisinsight/ui/src/pages/workbench/components/enablement-area/EnablementArea/EnablementArea.tsx:29]: if 5 arguments exists, I think should refact to 1 object argument `{...}`

File: redisinsight/ui/src/pages/workbench/components/enablement-area/EnablementArea/components/CodeButton/CodeButton.tsx (hunk 1 above)
Comment (romansergeenkosofteq) [reply redisinsight/ui/src/pages/workbench/components/enablement-area/EnablementArea/components/CodeButton/CodeButton.tsx]: fixex

File: redisinsight/ui/src/pages/workbench/components/enablement-area/EnablementArea/EnablementArea.tsx (hunk 2 above)
Comment (romansergeenkosofteq) [reply redisinsight/ui/src/pages/workbench/components/enablement-area/EnablementArea/EnablementArea.tsx:29]: fixed

**Review by zalenskiSofteq — APPROVED** [review]

**Pull request merged**